scripts/gen_bytecode.lua

```diff
@@ -43,7 +43,11 @@ function gen_bytecode()
         from_balance, from_seq = get_account(from)
         payload = sig_payload(to, value, sequence)
         check_sig(from, sig, payload)
-        if sequence < from_seq then
+	
+	hash = make_hash("Hello World!")
+	print(hash)      
+
+	if sequence < from_seq then
```

Comment:
Presumably, this is meant to test that the newly-exposed function works; can this be verified using a real test instead of adding IO to the bytecode generation?

On that note, when testing the exposure of a hash-function (or any data-transformation) specifically, it's probably a good idea to verify that it works against known values (i.e., I'd suggest ``assert``ing that the hash from ``"Hello World!"`` is what you expect).